scripts/play_vk

```diff
@@ -28,7 +28,7 @@ trap 'kill -s INT $PID' TERM INT
 
 # play file on the default sound device in backgroudn, so signals can be catch
 # by the 'trap' instruction above
-play $1 -q 2>/dev/null &
+play -q $1 2>/dev/null &
```

Comment:
Locally I found that redirecting `stderr` to `/dev/null` was not necessary with the `-q` option properly placed.  Please test that, Tom.

Reply:
> Sorry I missed this.
> 
Just found the following in 'man play':

"These options can be specified on the command line at any point  before the first effect name."

That may explain the behavior. With -q before the file name it is quite more robust. Thanks.

Reply:
> Locally I found that redirecting `stderr` to `/dev/null` was not necessary with the `-q` option properly placed. Please test that, Tom.

It may be left over from former versions of SoX which left an error message when killed. Just tested it with 'aplay -q' - that would need the redirection.

Reply:
I was basing my placement of `-q` on the output of `play -h` which provides a usage output.  Yes it says that gopts can be specified any place before the first effect.  Earlier in the output is this line:

```
Usage summary: [gopts] [[fopts] infile]... [fopts] [effect [effopt]]...
```

So I presumed it would be best to place the `-q` in front of the filename.  However, just testing it now I see it is honored in either place on the command line, so disregard my "advice".  :)

Reply:
I just tested the aplay command. But as agreed let us stay with SoX as 'official' supported.

And sure there has to be more documentation. At the moment it is just to test that the sound gets stopped and rig is going back to RX mode.

Reply:
I am going through the man page and have noted that area as one that will need to be updated.